src/stirling/source_connectors/socket_tracer/testing/container_images/BUILD.bazel

```diff
@@ -0,0 +1,368 @@
+        ],
+        deps = ["//src/common/testing/test_utils:cc_library"],
+    )
+    for sdk_version in pl_supported_go_sdk_versions
+]
+
+[
+    pl_cc_test_library(
+        name = "go_%s_tls_server_container" % sdk_version.replace(".", "_"),
+        srcs = [],
+        hdrs = ["go_%s_tls_server_container.h" % sdk_version.replace(".", "_")],
+        data = [
+            "//src/stirling/testing/demo_apps/go_https/server:golang_%s_https_server.tar" % sdk_version.replace(".", "_"),
+        ],
+        deps = ["//src/common/testing/test_utils:cc_library"],
+    )
+    for sdk_version in pl_supported_go_sdk_versions
+]
+
+pl_cc_test_library(
+    name = "golang_sqlx_container",
+    srcs = [],
+    hdrs = ["golang_sqlx_container.h"],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers/pgsql:demo_client_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "kafka_container",
+    srcs = [],
+    hdrs = ["kafka_container.h"],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers:kafka_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "mysql_container",
+    srcs = [],
+    hdrs = ["mysql_container.h"],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers:mysql_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "nats_client_container",
+    srcs = [],
+    hdrs = ["nats_client_container.h"],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/protocols/nats/testing:nats_test_client_with_ca_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "nats_server_container",
+    srcs = [],
+    hdrs = ["nats_server_container.h"],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers:nats_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "nginx_open_ssl_1_1_0_container",
+    srcs = [],
+    hdrs = [
+        "nginx_open_ssl_1_1_0_container.h",
+        "nginx_worker_pid.h",
+    ],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers:nginx_openssl_1_1_0_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "nginx_open_ssl_1_1_1_container",
+    srcs = [],
+    hdrs = [
+        "nginx_open_ssl_1_1_1_container.h",
+        "nginx_worker_pid.h",
+    ],
+    data = [
+        "//src/stirling/source_connectors/socket_tracer/testing/containers:nginx_openssl_1_1_1_image.tar",
+    ],
+    deps = ["//src/common/testing/test_utils:cc_library"],
+)
+
+pl_cc_test_library(
+    name = "nginx_open_ssl_3_0_7_container",
```

Comment:
We have a precedent for naming openssl as one word. I would prefer we keep that consistency.

Reply:
Done.
I also renamed the nginx_open_ssl_3_0_7 container/headers to `nginx_openssl_3_0_8` since when I inspected the container, it was running openssl 3.0.8

```
$ docker run --rm -it --entrypoint /bin/sh gcr.io/pixie-oss/pixie-dev-public/docker-deps/library/nginx@sha256:3eb380b81387e9f2a49cb6e5e18db016e33d62c37ea0e9be2339e9f0b3e26170
/ # strings /lib/libcrypto.so.3 | grep -o "OpenSSL [[:digit:]][^ ]*"
OpenSSL 3.0.8
```